zebra-network/src/meta_addr/arbitrary.rs

```diff
@@ -17,9 +19,31 @@ impl MetaAddr {
             .boxed()
     }
 
-    pub fn alternate_node_strategy() -> BoxedStrategy<Self> {
+    /// Create a strategy that generates [`MetaAddr`]s in the
+    /// [`PeerAddrState::NeverAttemptedAlternate`] state.
+    pub fn alternate_strategy() -> BoxedStrategy<Self> {
+        (canonical_socket_addr(), any::<PeerServices>())
+            .prop_map(|(socket_addr, services)| MetaAddr::new_alternate(&socket_addr, &services))
+            .boxed()
+    }
```

Comment:
Fix the alternate strategy so it returns the full range of possible service bits. (Node and client.)